### PR TITLE
Add add-messages-1 to master

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, coaches can not view messages or see requests for analysis.
- Currently, the `zombie` view is very cluttered and you can harldy distinguish between open and closed `zombies`.
- Currently, there is no version number.

**What is the new behavior (if this is a feature change)?**
- Coaches can now view messsenges and see requests for analysis. This includes a wide range of features, like marking one as closed.
- `Zombies` marked done now have a green `ResultChart`.
- The version nummber is now logged via console.

**Other information**:
- The version has been updated. (Minor)

### CODE REVIEW

**Review by sweggerat — APPROVED** [review]
looks good

**Review by LucaAllmaier — APPROVED** [review]
Looks good to me.

File: src/components/molecules/Footer/index.jsx [hunk 1]
```diff
@@ -1,12 +1,10 @@
 //> React
 // Contains all the functionality necessary to define React components
 import React from "react";
-// Router
-import { Link } from "react-router-dom";
 
 //> MDB
 // "Material Design for Bootstrap" is a great UI design framework
-import { MDBFooter, MDBRow, MDBCol, MDBContainer, MDBIcon } from "mdbreact";
+import { MDBFooter,MDBContainer, MDBIcon } from "mdbreact";
```
Comment (kleberbaum) [inline src/components/molecules/Footer/index.jsx]: missing whitespace after comma.

File: src/components/organisms/Requests/index.jsx [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    this.props.getData(false);
+  };
+
+  componentWillReceiveProps(nextProps) {
+    // Check if requests have changed
+    if (
+      JSON.stringify(this.props.contact) !== JSON.stringify(nextProps.contact)
+    ) {
+      nextProps.contact &&
+        this.setState({ sync: false });
```
Comment (kleberbaum) [inline src/components/organisms/Requests/index.jsx]: Please check indentation.

File: src/components/organisms/Requests/index.jsx [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+        </div>
+        {this.state.sync && (
+          <div>
+            <MDBProgress material preloader />
+          </div>
+        )}
+        {!this.state.sync &&
+        contact &&
+        contact.data &&
+        contact.data.length > 0 ? (
```
Comment (kleberbaum) [inline src/components/organisms/Requests/index.jsx:96]: Please check indentation.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: src/components/organisms/Requests/index.jsx (hunk 3 above)
Comment (Aichnerc) [reply src/components/organisms/Requests/index.jsx:96]: Indentation is okay.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

File: src/components/molecules/charts/ResultChart/index.jsx [hunk 4]
```diff
@@ -73,12 +73,14 @@ class ResultChart extends React.Component {
...
     if (!this.state.dataRadar) {
-      this.getChart();
+      this.getChart(green);
     }
   };
 
-  getChart = () => {
+  getChart = (green) => {
     if (this.props.data) {
       let labels, data;
```
Comment (schettn) [inline src/components/molecules/charts/ResultChart/index.jsx:85]: add newline

File: src/components/organisms/FormCat/index.jsx [hunk 5]
```diff
@@ -222,14 +211,12 @@ class FormCat extends React.Component {
...
 
-      console.log(userobject);
-
       this.props.createCat(userobject);
     }
   };
 
   render() {
-    const { auth, profile, catDetails } = this.props;
+    const { catDetails } = this.props;
```
Comment (schettn) [inline src/components/organisms/FormCat/index.jsx:219]: add newline

**Review by schettn — CHANGES_REQUESTED** [review]
Should be ok but please check my comments.

**Review by schettn — APPROVED** [review]
LGTM! 🦺

**Pull request merged**